react-function-components/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const container = document.getElementById('root');
+const root = ReactDOM.createRoot(container);
+
+root.render(CustomButton());
```

Comment:
With components, we never directly call them. To use a component we use JSX syntax:
```JSX
root.render(<CustomButton />);
```